packages/formik/src/Formik.tsx

```diff
@@ -416,7 +416,7 @@ export function useFormik<Values extends FormikValues = FormikValues>({
       if (props.onReset) {
         const maybePromisedOnReset = (props.onReset as any)(
           state.values,
-          imperativeMethods
+          imperativeMethodsAndState
```

Comment:
I would combine them here, like `{ ...imperativeMethods, ...state }`, instead of doing it on every render.